src/NServiceBus.AcceptanceTests/ScenarioDescriptors/AllTransports.cs

```diff
@@ -77,6 +77,19 @@ public AllTransportsWithoutNativeDeferral()
         }
     }
 
+    public class AllTransportsWithoutNativeDeferralAndAtomicSendAndReceive : AllTransports
```

Comment:
If I understand the logic of this filter correctly, I think a more descriptive name for this class would be

`AllTransportsWithoutNativeDeferralAndWithAtomicSendAndReceive`